pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -113,6 +113,7 @@ public SuccessResponse addTable(String tableConfigStr) {
     try {
       tableConfig = JsonUtils.stringToObject(tableConfigStr, TableConfig.class);
       TableConfigUtils.validate(tableConfig);
+      TableConfigUtils.validateTableName(tableConfig);
```

Comment:
Probably good to add some comments on why only validate it here

Reply:
added.